docs/en/FAQ/README.md

```diff
@@ -0,0 +1,3 @@
+## FAQ
+
+1. [How do we run Satellite on Windows?](./running_on_windows.md)
```

Comment:
Having an individual doc for building Windows is troublesome, please review https://github.com/apache/skywalking-satellite/pull/24 and rework this part base on that